packages/perseus/src/widgets/table/score-table.ts

```diff
@@ -41,12 +32,13 @@ function scoreTable(
             message: null,
         };
     }
+
     const createValidator = KhanAnswerTypes.number.createValidatorFunctional;
-    let message = null;
-    const allCorrect = _.every(solution, function (rowSolution) {
+    let message: string | null = null;
+    const allCorrect = solution.every(function (rowSolution) {
         for (let i = 0; i < supplied.length; i++) {
             const rowSupplied = supplied[i];
-            const correct = _.every(rowSupplied, function (cellSupplied, i) {
+            const correct = rowSupplied.every(function (cellSupplied, i) {
                 const cellSolution = rowSolution[i];
                 const validator = createValidator(
```

Comment:
Random thought unrelated to the PR: is calling this "validator" still correct? Or is it a scorer now? Or is it both still?

Reply:
This validator is related to khanAnswerTypes I believe, not the scoring/validation logic we worked on. I think we decided not to update its name, though I wonder if it might still lead to confusion though.